Dockerfile

```diff
@@ -86,36 +89,34 @@ WORKDIR /
 COPY --from=pgsql-stage /usr/lib/postgresql /usr/lib/postgresql
 COPY --from=pgsql-stage /usr/share/postgresql /usr/share/postgresql
 
-## Copy from dashboard-stage
-WORKDIR /dashboard
-COPY --from=dashboard-stage /dashboard/*.* ./
-COPY --from=dashboard-stage /dashboard/src src
-COPY --from=dashboard-stage /dashboard/public public
-COPY --from=dashboard-stage /dashboard/node_modules node_modules
+## Copy files
+COPY ./src/scripts/postgresql.conf /etc/postgresql/14/main/postgresql.conf
+COPY ./src/scripts/pg_hba.conf /etc/postgresql/14/main/pg_hba.conf
+COPY docker-entrypoint.sh /engine/docker-entrypoint.sh
+COPY src/scripts /engine/src/scripts
 
 ## Copy from engine-stage
 WORKDIR /engine
-COPY --from=engine-stage /engine/package.json ./
 COPY --from=engine-stage /engine/node_modules node_modules
+COPY --from=engine-stage /engine/package.json ./
 COPY --from=engine-stage /engine/dist dist
 
-## Copy files
-COPY ./src/scripts/postgresql.conf /etc/postgresql/14/main/postgresql.conf
-COPY ./src/scripts/pg_hba.conf /etc/postgresql/14/main/pg_hba.conf
-COPY docker-entrypoint.sh /engine/docker-entrypoint.sh
-COPY src/scripts /engine/src/scripts
+## Copy from dashboard-stage
+WORKDIR /dashboard
+COPY --from=dashboard-stage /dashboard/public ./public
+COPY --from=dashboard-stage /dashboard/.next/standalone ./
+COPY --from=dashboard-stage /dashboard/.next/static ./.next/static
 
 ## Default ENVs that can be overwritten
 ARG IASQL_ENV=local
 ENV IASQL_ENV=$IASQL_ENV
-ENV NEXT_PUBLIC_IASQL_ENV=$IASQL_ENV
 ARG IASQL_TELEMETRY=on
 ENV IASQL_TELEMETRY=$IASQL_TELEMETRY
-ENV NEXT_PUBLIC_IASQL_TELEMETRY=$IASQL_TELEMETRY
 ARG DB_USER=postgres
 ENV DB_USER=$DB_USER
 ARG DB_PASSWORD=test
 ENV DB_PASSWORD=$DB_PASSWORD
+ENV PORT=9876
```

Comment:
Why can't the `start:docker` in the dashboard package.json define this?

Reply:
since we are now running the prod version as a node app we cannot pass the port forwarding there, so we defined it as an env var here otherwise it took the default 3000. Do you want me to change the default one instead?

Reply:
No, I mean you could just have:


```json
"start:docker": "PORT=9876 node server.js",
```

To encapsulate that config there?